src/main.js

```diff
@@ -59,8 +49,7 @@ function randomCover(){
     var randomDescOne = descriptors[getRandomIndex(descriptors)]
     var randomCover = covers[getRandomIndex(covers)]
     var randomTitle = titles[getRandomIndex(titles)]
-   currentCover([randomTitle, randomCover, randomDescOne, randomDescTwo])
-   //savedCovers.push(randCover);
+   makeCurrentCover([randomTitle, randomCover, randomDescOne, randomDescTwo])
```

Comment:
I would suggest switching the first two elements in this array to make it consistent with the order of Cover class.  We'll also have to edit the makeCurrentCover function accordingly

Reply:
Went and edited per your recommendations. Take a look and see what you think!